shelley/chain-and-ledger/formal-spec/update.tex

```diff
@@ -0,0 +1,396 @@
+\section{Updates}
+\label{sec:update}
+
+
+The $\mathsf{UPDATE}$ transition is responsible for federated governance model in Shelly.
+This includes the ability to change the protocol parameters and also handles software
+updates.
+
+The application versions behave much like a protocol parameter, but because adoption
+for them must occur as soon as consensus is met, they are handled separately.
+Therefore the $\mathsf{UPDATE}$ is the combination of the
+Therefore the $\mathsf{PPUP}$ transition for the protocol parameters and the
+the $\mathsf{AVUP}$ transition for the application versions.
+
+Note that the signature for the keys in the proposal will be checked in the
+$\mathsf{UTXOW}$ transition.
+
+
+\begin{figure}[htb]
+  \emph{Protocol Parameter Update environment}
+  \begin{equation*}
+    \PPUpdateEnv =
+    \left(
+      \begin{array}{r@{~\in~}lr}
+        \var{slot} & \Slot & \text{current slot}\\
+        \var{dms} & \VKeyGen\mapsto\VKey & \text{genesis key delegations} \\
+      \end{array}
+    \right)
+  \end{equation*}
+  %
+  \emph{Protocol Parameter Update transitions}
+  \begin{equation*}
+    \_ \vdash
+    \var{\_} \trans{ppup}{\_} \var{\_}
+    \subseteq \powerset (\PPUpdateEnv \times \PPUpdate \times \PPUpdate \times \PPUpdate)
```

Comment:
Would it make sense to include a hyperlinked reference here to where `\PPUpdate` is defined?